clickable.py

```diff
@@ -1273,18 +1273,29 @@ def _build(self):
     def post_make(self):
         super(CordovaClickable, self).post_make()
 
-        copies = [
-            'www',
-            'config.xml',
-            'cordova.desktop',
-            'manifest.json',
-            'apparmor.json'
-        ]
+        www_dir = os.path.join(self.platform_dir, "www")
+        shutil.rmtree(www_dir)
+        shutil.copytree(os.path.join(self.cwd, "www"), www_dir)
```

Comment:
Is this part redundant because of the next few lines?

Reply:
Cordova does something weird with copying `$PROJECT_DIR/www` -> `$PROJECT_DIR/platforms/ubuntu/www` -> `$PROJECT_DIR/platforms/ubuntu/ubuntu-sdk-15.04/armhf/build/www`, and I'm attempting to keep the `platforms` dir structure the same.

The cordova code that this line is replicating can be found [here](https://github.com/apache/cordova-ubuntu/blob/658aac2f9066ae3a98a858e142504519b40c6165/bin/lib/update.js#L34-L36)

I would like your thoughts on a few alternatives:
1) symlink `$PROJECT_DIR/www` to `$PROJECT_DIR/platforms/ubuntu/www`. I decided against this because it wasnt *exactly* what cordova was doing, and I was worried there might have been some unintended behaviour if using the `cordova` command manually.
2) just run the javascript file [update.js](https://github.com/apache/cordova-ubuntu/blob/master/bin/lib/update.js) using node, which introduces an additional dependency, or could spool up in docker container (more complications). The benefit to this is it does exactly what cordova is expecting.
3) We could just ignore the cordova file structure and copy `$PROJECT_DIR/www` to `$PROJECT_DIR/platforms/ubuntu/ubuntu-sdk-15.04/armhf/build/www`, which could probably be simpler but would introduce the potential of any future `cordova` commands to fail because the internal structure of `platforms/ubuntu/` would be a bit different than it should be.

Reply:
Currently, solely doing this is unacceptable. Another file that is being depended on being copied in the same way is `config.xml`, so `$PROJECT_DIR/config.xml` is copied to `$PROJECT_DIR/platforms/ubuntu/config.xml` and then to `$PROJECT_DIR/platforms/ubuntu-ubuntu-sdk-15.04/armhf/build/config.xml`.

My current fix is to delete the platforms directory and let cordova rebuild it